common/djangoapps/student/tests/test_verification_status.py

```diff
@@ -271,13 +271,11 @@ def _assert_course_verification_status(self, status):
         # Sanity check: verify that the course is on the page
         self.assertContains(response, unicode(self.course.id))
 
-        # Verify that the correct banner is rendered on the dashboard
-        self.assertContains(response, self.BANNER_ALT_MESSAGES[status])
-
         # Verify that the correct banner color is rendered
         self.assertContains(
             response,
-            "<article class=\"course {}\">".format(self.MODE_CLASSES[status])
+            "<div role=\"region\" aria-labelledby=\"{course_id}\" class=\"course {status}\">".format(
```

Comment:
@ahsan-ul-haq ```{course_id}``` does not result in an IDref that exists.  It would need to point to an ID on the <h3> element that has the course title in it.  You can also just do this: 
```aria-label=\"{course_name}\"``` 
instead of 
```aria-labelledby=\"{course_id}\"```
I think that will be easier and have a better result